apps/agreement/contracts/Agreement.sol

```diff
@@ -773,16 +775,23 @@ contract Agreement is IAgreement, AragonApp {
             return;
         }
 
-        // Get staking pool
-        Staking staking = stakingFactory.getOrCreateInstance(token);
-
-        // Pull required fee amount from staking pool
-        _lockBalance(staking, _submitter, amount);
-        _slashBalance(staking, _submitter, address(this), amount);
+        uint256 sentEth = 0;
```

Comment:
Why do we need this?

Reply:
Simply to support both ETH or ERC20 fees

Reply:
But can’t you use `msg.value` everywhere?

Reply:
True 😅